pkg/linguist/linguist.go

```diff
@@ -11,7 +11,6 @@ import (
 
 	"github.com/Azure/draft/pkg/osutil"
 	log "github.com/Sirupsen/logrus"
-	"github.com/generaltso/linguist"
```

Comment:
how come we're removing the linguist package here?

Reply:
The main reason here is because of [the setup steps](https://github.com/generaltso/linguist#install) required to run generaltso/linguist. When you run `make`, it performs `go generate` to perform a `git` submodule checkout of github/linguist, runs a naive bayesian classifier against the samples which then generates the resultant Go code. I've taken that package and put it in pkg/linguist so developers won't have to do that any more.

Essentially it allows us to remove [this](https://github.com/Azure/draft/blob/5937caa7c2e678c25961b7d5699bea6fc2aca6e4/scripts/setup-apimachinery.sh#L27-L35) piece of hacky code. In its place, I've added a README to pkg/linguist so others know how to update pkg/linguist when we need to re-train the bayesian classifier against a newer version of github/linguist.

Reply:
yeah that makes sense! thank you